### PR TITLE
export cosmos collections straight from Prod to target

### PR DESCRIPTION
**Pull request by Raph-zek**
This to try and export cosmos collections straight from Prod DB into a target environment

### CODE REVIEW

File: PSScripts/Export-import-CosmosDbCollections.ps1 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    [Parameter(Mandatory=$false)]
+    [string]$CosmosKeyTarget,
+    [Parameter(Mandatory=$true)]
+    [string]$DataMigrationToolLocation = 'C:\Program Files (x86)\AzureCosmosDBDataMigrationTool\dt.exe'
+)
+
+foreach ($collection in $Collections) {
+    Write-Verbose -Message "Backing up collection $collection"
+
+  ##  $parameters = "/s:DocumentDB /s.ConnectionString:AccountEndpoint=https://$CosmosAccountName.documents.azure.com:443/;AccountKey=$CosmosKey;Database=$Database /s.Collection:$Collection /t:JsonFile /t.File:blobs://$StorageKey@$($ContainerUrl.Replace('https://', ''))/$([DateTime]::Now.ToString("yyyy-MM-dd_HHmm"))-$Collection-backup.json"
```
Comment (daniel-m-wise) [inline PSScripts/Export-import-CosmosDbCollections.ps1:26]: Can we remove the commented line if no longer needed?

**Review by Raph-zek — COMMENTED** [review]
First upload

File: PSScripts/Export-import-CosmosDbCollections.ps1 [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    [Parameter(Mandatory=$true)]
+    [string]$CosmosAccountName,
+    [Parameter(Mandatory=$false)]
+    [string]$Database,
+    [Parameter(Mandatory=$false)]
+    [array]$Collections,
+    [Parameter(Mandatory=$true)]
+    [string]$CosmosKey,
+    [Parameter(Mandatory=$true)]
+    ##[string]$StorageKey,
```
Comment (daniel-m-wise) [inline PSScripts/Export-import-CosmosDbCollections.ps1:12]: Remove Comment

File: PSScripts/Export-import-CosmosDbCollections.ps1 [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    ##[Parameter(Mandatory=$false)]
```
Comment (daniel-m-wise) [inline PSScripts/Export-import-CosmosDbCollections.ps1:13]: Remove Comment

**Review by daniel-m-wise — CHANGES_REQUESTED** [review]

**Review by daniel-m-wise — APPROVED** [review]

**Pull request merged**